client/package.json

```diff
@@ -7,9 +7,11 @@
     "babel-eslint": "^8.2.3",
     "bootstrap": "^4.1.1",
     "eslint-config-airbnb": "^16.1.0",
+    "history": "^4.7.2",
```

Comment:
I'm not sure if this was added accidentally. it looks like `history` from `react-router` is used properly in other parts of components (like Settings and SignUp. **Would recommend removing this pkg**.